Makefile.inc

```
@@ -15,6 +15,10 @@ else ifneq (,$(filter wireguard,$(_using)))
 override DEPLOY_ARGS += --cable_driver wireguard
 endif
 
+ifneq (,$(filter lighthouse,$(_using)))
+override DEPLOY_ARGS += --deploytool_broker_args '--service-discovery'
+endif
```

Service discovery is enabled by default, don’t we need

```suggestion
ifneq (,$(filter lighthouse,$(_using)))
override DEPLOY_ARGS += --deploytool_broker_args '--service-discovery'
else
override DEPLOY_ARGS += --deploytool_broker_args '--service-discovery=false'
endif
```

instead?